### PR TITLE
MJ-28 added Identity schema and essential Razor pages.

### PR DESCRIPTION
**Pull request by juliapacula**

### CODE REVIEW

File: server/Mistakes.Journal.Api/Api/Mistakes/Controllers/MistakesController.cs [hunk 1]
```diff
@@ -63,10 +69,6 @@ public async Task<ActionResult<MistakeWebModel>> AddMistake(NewMistakeWebModel n
         public async Task<ActionResult<MistakeWebModel>> CloneMistake(Guid mistakeId)
         {
             var mistake = await _dataContext.Set<Mistake>()
-                .Include(m => m.Tips)
-                .Include(m => m.MistakeLabels)
-                    .ThenInclude(m => m.Label)
-                .Include(m => m.Repetitions)
```
Comment (PawelKriger) [inline server/Mistakes.Journal.Api/Api/Mistakes/Controllers/MistakesController.cs]: Just to make sure: after cloning, won't new mistake have empty fields?

File: server/Mistakes.Journal.Api/Api/Mistakes/Controllers/MistakesController.cs [hunk 2]
```diff
@@ -85,9 +87,6 @@ public async Task<ActionResult<MistakeWebModel>> AddRepetitionResetCounter(Guid
 
             var mistake = await _dataContext.Set<Mistake>()
                 .Include(m => m.Repetitions)
-                .Include(m => m.MistakeLabels)
-                    .ThenInclude(m => m.Label)
-                .Include(m => m.Tips)
```
Comment (PawelKriger) [inline server/Mistakes.Journal.Api/Api/Mistakes/Controllers/MistakesController.cs]: here also

File: server/Mistakes.Journal.Api/Api/Mistakes/Controllers/MistakesController.cs (hunk 1 above)
Comment (juliapacula) [reply server/Mistakes.Journal.Api/Api/Mistakes/Controllers/MistakesController.cs]: you were right, fixed

File: server/Mistakes.Journal.Api/Api/Mistakes/Controllers/MistakesController.cs (hunk 2 above)
Comment (juliapacula) [reply server/Mistakes.Journal.Api/Api/Mistakes/Controllers/MistakesController.cs]: you were right, fixed

**Review by PawelKriger — APPROVED** [review]

**Pull request merged**